crates/ruma-events/tests/it/room_message.rs

```diff
@@ -599,6 +601,67 @@ fn file_msgtype_encrypted_content_deserialization() {
     assert_eq!(encrypted_file.url, "mxc://notareal.hs/file");
 }
 
+#[test]
```

Comment:
Could you add tests for serialization and deserialization of a custom `itemtype`? Just to make sure that it works and that we don't break it accidentally.

Reply:
Good call. Have add two tests for a custom itemtype.